example/tkg-k8s-block-driver/example-static-block-provisioning.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: PersistentVolume
```

Comment:
Why do we need 2 examples - one for TKGS and another for vanilla?

Reply:
I had initially created the vanilla static provisioning PVC without using storage class. Will change it to using storage class and mention PV name. I will also remove the tkgs example. Thanks for pointing this out.